com.vinszent.GnomeTwitch.json

```diff
@@ -0,0 +1,175 @@
+    "branch": "stable",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "gnome-twitch",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
+    "cleanup": [
+        "*.la",
+        "*.a",
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/bash-completion",
+        "/share/gtk-doc",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "ninja",
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/ninja-build/ninja.git",
+                    "branch": "v1.7.1"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "configure",
+                    "commands": [
+                        "./configure.py --bootstrap"
+                    ]
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "Makefile",
+                    "commands": [
+                        "all:",
+                        "install:",
+                        "\tinstall -D ninja /app/bin/ninja"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "meson",
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mesonbuild/meson.git",
+                    "branch": "0.33.0"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "configure",
+                    "commands": [ "" ]
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "Makefile",
+                    "commands": [
+                        "all:",
+                        "install:",
+                        "\tpython3 ./setup.py install --prefix=/app/"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "yasm",
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
+                    "sha256": "3dce6601b495f5b3d45b59f7d2492a340ee7e84b5beca17e48f862502bd5603f"
+                }
+            ]
+        }, {
+            "name": "gst-libav",
```

Comment:
Could be in a separate package as well. See https://github.com/flatpak/flatpak/issues/123